pkgdb/src/realisepkgs/realise.cc

```diff
@@ -349,21 +363,12 @@ std::vector<std::pair<realisepkgs::RealisedPackage, nix::StorePath>>
 getRealisedOutputs( nix::ref<nix::EvalState> &       state,
                     const RealisepkgsLockedPackage & lockedPackage )
 {
-  debugLog( nix::fmt( "getting cursor for %s", lockedPackage.attrPath[0] ) );
   auto timeEvalStart = std::chrono::high_resolution_clock::now();
-  auto cursor        = evalCacheCursorForInput( state,
-                                         lockedPackage.input,
-                                         lockedPackage.attrPath );
-
   /**
    * Collect the store paths for each output of the package.
    * Note that the "out" output is the same as the package's outPath.
    */
-  auto outputsToOutpaths
-    = outpathsForPackageOutputs( state, lockedPackage.installId, cursor );
-
-
-  auto pkgs = collectRealisedOutputs( state, lockedPackage, outputsToOutpaths );
+  auto pkgs        = collectRealisedOutputs( state, lockedPackage );
```

Comment:
note nonblocking: not really the fault of this PR, but I found this pretty difficult to review because names are kinda misleading (e.g. I don't think `getRealisedOutputs` actually realizes anything) and there aren't any comments. It seems like a `RealisedPackage` hasn't actually been realised(?). @ysndr may have more insight on some of that

Reply:
Same as comment below... would like to refactor this a bit now for a few reasons.